edx-platform/pearson-theme/lms/templates/courseware/course_about.html

```diff
@@ -271,8 +285,58 @@ <h2 class="opencourseware">MITOpenCourseware</h2>
       % endif
     %endif
   </div>
+  </div>
+  %if button_enroll_bottom:
+  <div class="main-cta button-enroll-bottom">
+    %if user.is_authenticated and registered:
+      %if show_courseware_link:
+        <a href="${course_target}">
+      %endif
 
+      <span class="register disabled">${_("You are enrolled in this course")}</span>
+
+      %if show_courseware_link:
+        <strong>${_("View Course")}</strong>
+        </a>
+      %endif
```

Comment:
This block of code could be reduced by a single if condition. @amalbas

Reply:
That is the code of the platform, and is in two parts because not always the should appear the link.